### ISSUE TITLE
pipeline deployment issue [LighthouseImageTags failed]

### ISSUE THREAD

**Issue by KirbyFox** [Shippable/support#2674]
Hello,

I am having an issue deploying a pipeline that i have deployed successfully before.

when i click the "deploy" button in the "configuration" I get this popup error "Deployment Failed: async Series check for LighthouseImageTags failed".

I tried to search online to see if anyone had this issue but i could not find any resources to resolve this issue.

Any help would be great. This is a time sensitive issue.
Thank you

*Screenshot below
![capture](https://cloud.githubusercontent.com/assets/18433298/15757877/9ab3892e-28bc-11e6-942e-9463e7797a45.JPG)

**Comment by a-murphy** [comment]
The image tag that you are trying to deploy doesn't appear to exist.  Has it been deleted on Docker Hub?

To deploy, first go to the pipeline settings, click "New Version," select a new tag for your image, and click "create" to save it.  Then go to the pipeline in that environment and deploy the new manifest version.

**Comment by KirbyFox** [comment]
Hi a-murphy,

Thanks for the quick reply.

I just checked the tags following your instructions. I don't see the tag that was linked to this pipeline configuration anymore in the dropdown, it was before there before. I see another tag I have used in the past in the dropdown but that tag is pointing to our production system and we are currently working from our development branch. I then checked Docker Hub to verify that the tag we needed was not modified, it was not modified. I then tried clicking the "Sync" button to refresh the tags in the pipeline configuration area, but it doesn't seem to be pulling the tag i need from Docker Hub.
